models/response.js

```diff
@@ -0,0 +1,56 @@
+var User = require('./user');
+var Service = require('./request');
+module.exports = function(sequelize, DataTypes) {
+    var Response = sequelize.define("Response", {
```

Comment:
I'm realizing it will make more sense if we pluralize the model names since they're synonymous with table names.

So let's change both instances of `Response` to `Responses` here.

It makes more sense with the Sequelize method naming conventions like `Responses.findOne()` and `Responses.findAll()`